packages/office-ui-fabric-react/src/components/Facepile/Facepile.Props.ts

```diff
@@ -53,6 +53,11 @@ export interface IFacepileProps extends React.Props<Facepile> {
 
   /** Method to access properties on the underlying Persona control */
   getPersonaProps?: (persona: IFacepilePersona) => IPersonaProps;
+
+  /**
+ * Optional class for facepile.
+ */
+  className?: string;
```

Comment:
not sure if we should add className or change Facepile to extend React.HTMLAttributes. I guess in the end it won't matter. If we extend later, we can remove this as an explicit prop.